src/client/components/footer/privacy.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import type { FC } from 'react';
+
+const Privacy: FC = () => {
+  if (process.env.REACT_APP_EOP_PRIVACY) {
+    return (
+      <a href={process.env.REACT_APP_EOP_PRIVACY}>Privacy</a>
+    );
+    }
+    return (<></>);
```

Comment:
We can also just return `null` in this case. That would make more sense to me, than the empty fragment.

Reply:
done.